src/sql/sqlops.proposed.d.ts

```diff
@@ -375,6 +375,16 @@ declare module 'sqlops' {
 				 */
 				content: string | DialogTab[],
 
+				/**
+				 * Register model view content for the dialog
+				 */
+				registerContent(handler: (view: ModelView) => void): void;
+
+				/**
+				 * Returns he model view content
+				 */
+				modelView: ModelView;
```

Comment:
Should this be read only? As far as I know we don't have a good way of updating the model view if the user sets it themselves. I'm also not entirely sure how the user would get an actual ModelView object to set here